reward/staking_manager.go

```diff
@@ -235,6 +242,14 @@ func StakingManagerUnsubscribe() {
 	stakingManager.chainHeadSub.Unsubscribe()
 }
 
+// SetTestStakingManagerWithDB sets the staking manager with the given database.
+// Note that this method is used only for testing purpose.
+func SetTestStakingManagerWithDB(testDB stakingInfoDB) {
```

Comment:
Hmm.. is it okay to use like this? this seems for testing, but it is in real code?

Reply:
Okay, but the there is another function to set it for testing. I will move these methods to test file in another PR.

Reply:
If you already consider this in your mind, could you add "TODO" here not to forget?

Reply:
I have added TODO comment for this as well as protocol number.